### ISSUE TITLE
Crashes Inkdrop on macOS Catalina

### ISSUE THREAD

**Issue by Lanithane** [inkdropapp/inkdrop-simple-note-list#3]
I'm getting the following error when enabling this plugin on macOS (v10.15.7):

```
Invariant Violation: Minified React error #321; visit https://reactjs.org/docs/error-decoder.html?invariant=321 for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at ca (/Users/Z0027PF/node_modules/react/cjs/react.production.min.js:11:316)
    at B (/Users/Z0027PF/node_modules/react/cjs/react.production.min.js:12:166)
    at W (/Users/Z0027PF/node_modules/react/cjs/react.production.min.js:20:375)
    at useCallback (/Users/Z0027PF/node_modules/react/cjs/react.production.min.js:22:362)
    at SimpleNoteListItemView (/Users/Z0027PF/Library/Application Support/inkdrop/packages/simple-note-list/lib/note-list-item-view.js:58:45)
    at oh (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:153:152)
    at Rj (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:261:502)
    at Qj (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:246:271)
    at Kj (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:246:200)
    at yj (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:239:178)
    at /Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:123:121
    at exports.unstable_runWithPriority (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/scheduler/cjs/scheduler.production.min.js:19:473)
    at cg (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:122:331)
    at fg (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:123:67)
    at gg (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:122:502)
    at Mj (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-dom/cjs/react-dom.production.min.js:240:72)
    at Object.notify (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-redux/lib/utils/Subscription.js:25:13)
    at Subscription.notifyNestedSubs (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-redux/lib/utils/Subscription.js:98:26)
    at Subscription.handleChangeWrapper (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/react-redux/lib/utils/Subscription.js:103:18)
    at dispatch (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/redux/lib/redux.js:228:13)
    at /Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/redux-thunk/lib/index.js:14:22
    at file:///Applications/Inkdrop.app/Contents/Resources/app.asar/browser-main.js:4:559855
    at dispatch (/Applications/Inkdrop.app/Contents/Resources/app.asar/app/static/node_modules/redux/lib/redux.js:644:34)
    at file:///Applications/Inkdrop.app/Contents/Resources/app.asar/browser-commons.js:1:31816
    at async Promise.all (index 0)
```

**Comment by craftzdog** [comment]
Weird. I can't reproduce it.
Can you please try disabling other plugins?

**Comment by Lanithane** [comment]
@craftzdog
I did some digging and it's failing because Inkdrop was attempting to pull dependencies from a `node_modules` folder I have at the root of my filesystem (line two from the stack trace):
L2: `at ca (/Users/Z0027PF/node_modules/react/cjs/react.production.min.js:11:316)`

I removed my local copy of `node_modules` since I didn't actually need it, and the error went away, so it seems there's a potential issue with Inkdrop pulling dependencies outside of its own build?

**Comment by craftzdog** [comment]
That's interesting.
I installed `react` in my home directory and edit its `index.js` to throw an error, then it reproduced.
The app adds a search path like so:

```
require('module').globalPaths.push('path/to/internal/module')
```

But apparently, it does not look up the `globalPaths` first.
Hmm, I don't know how to solve it.

**Comment by craftzdog** [comment]
Sorry for the delay.
Fixed in v0.1.2!
